### PR TITLE
Added tipping

### PR DESCRIPTION
**Pull request by Astro-Choco**
For DEVX-385, copied the Documentation directory from payments-sdk-ios and updated README accordingly.

Please note that README is structured differently and contains some other diff compared to the iOS README at the moment.

### CODE REVIEW

**Comment by StefanKnorn** [comment]
There seems to be a duplication in SDK_Tipping_Support_Documentation.md   Rows 50 is duplicated further down.

**Comment by Astro-Choco** [comment]
> There seems to be a duplication in SDK_Tipping_Support_Documentation.md Rows 50 is duplicated further down.

Good catch, @StefanKnorn . I just checked. It has been there since before in iOS SDK repo. Maybe we should delete the duplication from all 4 repos. Please let me know your thoughts.

**Comment by StefanKnorn** [comment]
If there is a duplication in the same document I think it should be removed @Astro-Choco

**Comment by Astro-Choco** [comment]
> If there is a duplication in the same document I think it should be removed @Astro-Choco

Thanks for the confirmation. I will remove the duplication.

File: Documentation/SDK_Tipping_Support_Documentation.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Denmark
+- Finland
+- United Kingdom
+- The Netherlands
+- Germany
+
+### Additional material
+
+- Zettle SDK FAQ Tipping
+- Description of user interaction on Zettle reader when receiving tips
```
Comment (bobzettle) [inline Documentation/SDK_Tipping_Support_Documentation.md:37]: @Astro-Choco I don't see where this is referring to?

Comment (Astro-Choco) [reply Documentation/SDK_Tipping_Support_Documentation.md:37]: I think it meant to reference to the Zettle SDK FAQ Tipping section below and the PDFs in the Documentation folder. I understand your point. :) For usability, it'd better have hyperlinks. Let's fix them when fixing the Documentation folder in the near future.

**Comment by Astro-Choco** [comment]
@bobzettle I moved tipping PDFs to a subfolder. Please let me know if the hyperlinks work as you expected.

**Review by bobzettle — APPROVED** [review]

**Comment by bobzettle** [comment]
> @bobzettle I moved tipping PDFs to a subfolder. Please let me know if the hyperlinks work as you expected.

**Review by StefanKnorn — APPROVED** [review]
Good with me

**Pull request merged**